samples/csharp_dotnetcore/70.skills-simple-bot-to-bot/README.md

```diff
@@ -0,0 +1,27 @@
+# SimpleBotToBot (**DRAFT**)
+
+Bot Framework v4 Skills Echo sample.
+
+This bot has been created using [Bot Framework](https://dev.botframework.com), it shows how to use skills from a rootbot.
+
+## Prerequisites
+
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 2.1
+
+  ```bash
+  # determine dotnet version
+  dotnet --version
+  ```
+
+## Key concepts
+
+- A [SimpleRootBot](SimpleRootBot/README.md) that can consume an echo skill.
```

Comment:
I think these bullets are now stale. Perhaps replace with short descriptions of what a host and skill bot are, or link to the skills concept doc, once it's published.